prometheus_juju_exporter/collector.py

```diff
@@ -184,15 +217,15 @@ def _get_container_status(
         """
         for container in containers.values():
             value = self._get_gauge_value(container["agent-status"]["status"])
+            machine_id = self._get_machine_identifier(container)
```

Comment:
Sorry for back and forth @mert-kirpici, the `_get_machine_identifier` was my idea, but Tianqi might be right here, since we are also dealing with LXC containers

Reply:
done